### ISSUE TITLE
Help button: button element not an html input/button (1.1.1, 1.3.1, 2.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#861]
### Expected behavior

Help button is an html input or button element role with a text label/equivalent to communicate its purpose.
- Element is `<button>` or  `<input type="button"`
  - it can be reached and "clicked" with a keyboard
  - button role semantics communicate its function
  - it can receive focus and visually indicates when this occurs
- Help icon has text equivalent
### Actual behavior

Element is a div which can't be reached/used with a keyboard. There is no text equivalent for the help question mark icon.

**Comment by jackmakesthings** [comment]
Hi! I can work on this one.

**Comment by Blackbaud-PatrickOFriel** [comment]
Thanks! Let us know if you need any help

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.9.0](https://github.com/blackbaud/skyux/releases/tag/v1.9.0)
